etl/steps/data/garden/hmd/2022-11-04/life_tables.meta.yml

```diff
@@ -1,5 +1,5 @@
 dataset:
-  sources:
+  sources: []
```

Comment:
Is this the official way to say "Nothing new in metadata". I find it a bit ugly that for datasets it's {}, while for sources it's []. I suppose this is YAML complexity in action.

Reply:
It's {} for `dataset` because it contains keys and [] for sources because it's a list. I'm not sure why `sources` was there, I removed them and kept only `dataset: {}`. We could make `dataset:` optional and remove it too (and fix function for loading metadata). I don't mind either.